### PR TITLE
docs: add example for: awscc_cloudformation_stack_set

### PR DESCRIPTION
**Pull request by ravramch**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

File: templates/resources/cloudformation_stack_set.md.tmpl [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+### Basic StackSet Usage
+To Create a basic StackSet
+{{ tffile (printf "examples/resources/%s/simple_stackset.tf" .Name)}}
+
+{{ .SchemaMarkdown | trimspace }}
+{{- if .HasImport }}
+
+### Advanced StackSet Usage
+To Create a StackSet with Service Managed permissions 
+{{ tffile (printf "examples/resources/%s/service_managed_stackset.tf" .Name)}}
```
Comment (wellsiau-aws) [inline templates/resources/cloudformation_stack_set.md.tmpl:23]: do not add example below the `{{ .SchemaMarkdown }}`

File: examples/resources/awscc_cloudformation_stack_set/simple_stackset.tf [hunk 2]
```diff
@@ -0,0 +1,23 @@
+resource "awscc_cloudformation_stack_set" "stackset" {
+  stack_set_name = "network-stackset"
```
Comment (wellsiau-aws) [inline examples/resources/awscc_cloudformation_stack_set/simple_stackset.tf:2]: make sure to run terraform fmt and remove blank space between the attributes

File: examples/resources/awscc_cloudformation_stack_set/service_managed_stackset.tf [hunk 3]
```diff
@@ -0,0 +1,28 @@
+resource "awscc_cloudformation_stack_set" "stackset" {
+  stack_set_name = "network-stackset1"
+
```
Comment (wellsiau-aws) [inline examples/resources/awscc_cloudformation_stack_set/service_managed_stackset.tf:3]: run terraform fmt and remote empty space between attributes

File: examples/resources/awscc_cloudformation_stack_set/service_managed_stackset.tf [hunk 4]
```diff
@@ -0,0 +1,28 @@
+resource "awscc_cloudformation_stack_set" "stackset" {
+  stack_set_name = "network-stackset1"
```
Comment (wellsiau-aws) [inline examples/resources/awscc_cloudformation_stack_set/service_managed_stackset.tf:2]: is there reason this has to be named as `network-stackset1` ?

File: docs/resources/cloudformation_stack_set.md [hunk 5]
```diff
@@ -121,10 +148,43 @@ Required:
 - `key` (String) A string used to identify this tag. You can specify a maximum of 127 characters for a tag key.
 - `value` (String) A string containing the value for this tag. You can specify a maximum of 256 characters for a tag value.
 
+### Advanced StackSet Usage
+To Create a StackSet with Service Managed permissions
```
Comment (wellsiau-aws) [inline docs/resources/cloudformation_stack_set.md:152]: Please add call out to enable Organizations access before using service-managed permissions

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
thanks for the contribution, please review the requested changes

File: examples/resources/awscc_cloudformation_stack_set/service_managed_stackset.tf (hunk 4 above)
Comment (ravramch) [reply examples/resources/awscc_cloudformation_stack_set/service_managed_stackset.tf:2]: Not really. I was working with 2 examples with self managed and service managed and had to differentiate while testing. Will change it to `network-stackset`

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thank you

**Comment by marcosentino** [comment]
@ravramch Thank you for the contribution!

**Pull request merged**